src/commonmark/plugins/triple-click-handler.ts

```diff
@@ -0,0 +1,18 @@
+import { Plugin } from "prosemirror-state";
+
+/** Plugin that selects only the current node on triple click */
```

Comment:
The doc here is slightly misleading:

```suggestion
/** Plugin that restores browser default triple-click handling behavior */
```